db/migrate/20200309043703_add_column.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumn < ActiveRecord::Migration[6.0]
+  def change
+    add_column :treatments, :store_id, :int
```

Comment:
If you look at your migrations in order, the treatments table doesn't exist when this one gets executed so an error is thrown.